backtesting_graphs/_plotting.py

```diff
@@ -125,69 +100,32 @@ def _maybe_resample_data(resample_rule, df, indicators, equity_data, trades):
     from .lib import OHLCV_AGG, TRADES_AGG, _EQUITY_AGG
     df = df.resample(freq, label='right').agg(OHLCV_AGG).dropna()
 
-    indicators = [_Indicator(i.df.resample(freq, label='right').mean()
-                             .dropna().reindex(df.index).values.T,
-                             **dict(i._opts, name=i.name,
-                                    # Replace saved index with the resampled one
-                                    index=df.index))
-                  for i in indicators]
-    assert not indicators or indicators[0].df.index.equals(df.index)
-
     equity_data = equity_data.resample(freq, label='right').agg(_EQUITY_AGG).dropna(how='all')
     assert equity_data.index.equals(df.index)
-
-    def _weighted_returns(s, trades=trades):
-        df = trades.loc[s.index]
-        return ((df['Size'].abs() * df['ReturnPct']) / df['Size'].abs().sum()).sum()
-
-    def _group_trades(column):
-        def f(s, new_index=pd.Index(df.index.view(int)), bars=trades[column]):
-            if s.size:
-                # Via int64 because on pandas recently broken datetime
-                mean_time = int(bars.loc[s.index].view(int).mean())
-                new_bar_idx = new_index.get_loc(mean_time, method='nearest')
-                return new_bar_idx
-        return f
-
-    if len(trades):  # Avoid pandas "resampling on Int64 index" error
-        trades = trades.assign(count=1).resample(freq, on='ExitTime', label='right').agg(dict(
-            TRADES_AGG,
-            ReturnPct=_weighted_returns,
-            count='sum',
-            EntryBar=_group_trades('EntryTime'),
-            ExitBar=_group_trades('ExitTime'),
-        )).dropna()
-
-    return df, indicators, equity_data, trades
+    return df, equity_data, trades
 
 
 def plot(*, results: pd.Series,
          df: pd.DataFrame,
-         indicators: List[_Indicator],
          filename='', plot_width=None,
+         show_plot=False,
          plot_equity=True, plot_return=False, plot_pl=True,
          plot_volume=True, plot_drawdown=False,
-         smooth_equity=False, relative_equity=True,
+         relative_equity=True,
          superimpose=True, resample=True,
-         reverse_indicators=True,
          show_legend=True, open_browser=True):
-    """
-    Like much of GUI code everywhere, this is a mess.
-    """
-    # We need to reset global Bokeh state, otherwise subsequent runs of
-    # plot() contain some previous run's cruft data (was noticed when
-    # TestPlot.test_file_size() test was failing).
-    if not filename and not IS_JUPYTER_NOTEBOOK:
-        filename = _windos_safe_filename(str(results._strategy))
+    """ Plots all Bokeh figures. We modified this function from backtesting.py and
+        primarily changed the _plot_equity_section function to overlay the equity
+        graph with additional lines. """
```

Comment:
Should I be adding docstrings to the functions in these files? I know this is a bad doc, but I can change it if need be.